docs/workers.md

```diff
@@ -343,11 +343,18 @@ in the config file, via the `fs.auto_clean_trashed_after` parameter.
 
 ## share workers
 
-The stack have 3 workers to power the sharings (internal usage only):
+The stack have 4 workers to power the sharings (internal usage only):
 
-1. `share-track`, to update the `io.cozy.shared` database
-2. `share-replicate`, to start a replicator for most documents
-3. `share-upload`, to upload files
+1. `share-group`, to add/remove members to a sharing
```

Comment:
@sblaisot for your information, I plan to add a new worker called `share-group`

Reply:
:+1: